src/Facility.Core/ServiceDto.cs

```diff
@@ -10,7 +10,7 @@ public abstract class ServiceDto
 		/// <summary>
 		/// Returns the DTO as JSON.
 		/// </summary>
-		public override string ToString() => ServiceJsonUtility.ToJson(this);
+		public override string ToString() => ServiceSerializer.Default.ToString(this);
```

Comment:
Should there be an overload that takes a serializer?

Reply:
It shouldn't be that useful, but it could raise awareness.

Reply:
On the other hand, we shouldn't encourage its use. And then we'd probably want to add it to `ServiceResult`. We should probably just leave it like this.